Documentation/devicetree/bindings/arm/analog/adi,sc5xx.yaml

```
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/arm/analog/adi,sc5xx.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Analog Devices SC59X 64-Bit ARM-based Processor Families
```

```suggestion
title: Platforms based on Analog Devices SC59X 64-Bit SoCs
```
But the filename is `sc5xx.yaml`. So maybe `sc59x.yaml` or as much as I hate it, `sc59x_64.yaml` since aarch64 isn't in a separate directory here.

---

Actually come to think of it, looks like there is no EV-SC596-SOM. So this should just be SC598